frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts

```diff
@@ -115,6 +126,40 @@ const osUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
       { isHidden: asHidden(!isWindows, VMSettingsField.OPERATING_SYSTEM), value: isWindows },
     ),
   );
+
+  const relevantOptions = iGetRelevantTemplateSelectors(state, id);
```

Comment:
relevant template can change on relevenatOptions, not just os. IMO this needs its own updater and possibly new file called eg commonTemplatesPrefil

Reply:
as mentioned here https://bugzilla.redhat.com/show_bug.cgi?id=1874074.

We are going to need a generic approach to common template prefill soon.